src/pages/Login.jsx

```diff
@@ -1,5 +1,62 @@
+import { useState } from "react";
+import { useAuth } from "../contexts";
+import { loginHandler } from "../helper-functions";
+import { Link, useNavigate } from "react-router-dom";
+
 function Login() {
-  return <h1>Login</h1>;
+  const { boolFunc } = useAuth();
+  const navigate = useNavigate();
+  const [pass, setPass] = useState("gkool");
+  const [email, setEmail] = useState("gkool@neog.camp");
+
+  return (
+    <div className="center-hv">
+      <div className="auth-form">
+        <form
+          onSubmit={(e) => loginHandler(e, boolFunc, email, pass, navigate)}
+        >
+          <h2>Login</h2>
+          <div className="inp-unit">
+            <label for="email">Email</label>
+            <input
+              type="email"
+              name="email"
+              value="johnsangram@gkool.com"
```

Comment:
is the login created just a placeholder to get over the hump and use apis? 
this value means user wont be able to log in with their own credentials?